docs/how-to/hip_runtime_api/initialization.rst

```diff
@@ -0,0 +1,124 @@
+.. meta::
+   :description: Initialization.
+   :keywords: AMD, ROCm, HIP, initialization
+
+********************************************************************************
+Initialization
+********************************************************************************
+
+Initialization involves setting up the environment and resources needed for GPU
+computation. The following steps are covered with the initialization:
+
+- Initialize the HIP runtime
```

Comment:
Hm. This is kind of redundant - the initialization starts with the initialization, but "setting up" also doesn't quite fit, as the function is literally called `hipInit`
```suggestion
- Setting up the HIP runtime
```